src/lib/progress-spinner/progress-spinner.ts

```diff
@@ -103,6 +107,11 @@ export class MdProgressSpinner implements OnDestroy {
     this._cleanupIndeterminateAnimation();
   }
 
+  /** Stroke width of the progress spinner. By default uses 10px as stroke width. */
+  @Input()
```

Comment:
why not just use 

`@Input() strokeWidth: number = DEFAULT_STROKE_WIDTH;`

Then it will have the default value, otherwise will use the input bound value.